src/features/game/mode/talk/GameModeTalk.ts

```diff
@@ -51,17 +51,17 @@ class GameModeTalk implements IGameUI {
       maxYSpace: TalkModeConstants.button.ySpace
     });
 
+    let topicId = 0;
     talkMenuContainer.add(
       buttons.map((button, index) =>
         this.createTalkTopicButton(
-          button.text,
+          '[ ' + (topicId++ + 1) + ' ]  ' + button.text,
```

Comment:
After seeing this direct string concatenation more than once, I am starting to think it'll be a better idea to extract this out into a separate method (e.g. `addButtonInstruction`) elsewhere?